modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -33,26 +39,26 @@
      */
     private static final String ELASTICSEARCH_PASSWORD = "changeme";
 
-    private RestClient client = null;
+    private RestClient restClient = null;
```

Comment:
IMHO this is not needed. The transport client test only creates one local transport client.
From elasticsearch point of view, in the near future, a client is a Rest Client. The Transport Client is meant to be removed in the future.